apps/web/src/app/globals.css

```diff
@@ -1 +1,46 @@
 @layer reset, base, tokens, recipes, utilities;
+
+@layer reset {
+  @font-face {
+    font-family: 'DNFBitBitv2';
+    font-style: normal;
+    font-weight: 700;
+    src: url('//cdn.df.nexon.com/img/common/font/DNFBitBitv2.otf') format('opentype');
+  }
+
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
+  }
+
+  body {
+    font-family: 'DNFBitBitv2';
+    font-style: normal;
+    font-weight: 400;
+    line-height: 140%; /* 107.8px */
+    background-color: #2a7442;
+    position: relative;
+  }
+
+  button {
+    cursor: pointer;
+    border: transparent;
+    background-color: transparent;
+    font-family: 'DNFBitBitv2';
+    line-height: 1.5;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 140%;
+    padding: 0;
+  }
+
+  input {
+    font-family: 'DNFBitBitv2';
+
+    &::placeholder {
+      color: #b5b5b5;
+      font-family: 'DNFBitBitv2';
+    }
+  }
```

Comment:
css nesting은 scss와 같은 전처리기와 달리 브라우저에서 직접 파싱이 돼서,

아직 브라우저 호환성이 높지 않더라구요 😢 

![스크린샷 2024-06-28 오전 8 44 33](https://github.com/git-goods/git-animal-client/assets/26461307/3133033e-b046-4d2e-af91-8f6cb6eee077)

https://caniuse.com/?search=css%20nesting

```suggestion
  input {
    font-family: 'DNFBitBitv2';
  }
  
  input::placeholder {
    // ...
  }
```

요롷게 해보면 어떨까요 ~??